### PR TITLE
Add Google search index

### PR DESCRIPTION
**Pull request by ahmadeyamin**

### CODE REVIEW

**Review by hackermondev — CHANGES_REQUESTED** [review]
we dont support TXT record

File: domains/ahmadeyamin.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "email": "ahmadeyamin@gmail.com"
   },
   "record": {
-    "CNAME": "ahmadeyamin.github.io"
+    "CNAME": "ahmadeyamin.github.io",
+    "TXT": "google-site-verification=I8tY35CoSN7YltvlCbW-yp0BqHkb1DPu6gOrgnFlMVU"
```
Comment (TheBotlyNoob) [inline domains/ahmadeyamin.json:10]: We also don't support more than one record

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Comment by mtgsquad** [comment]
Hey, if you want to index your site on google, please use the meta tag verification, then you can simply add it to your sites HTML :)

**Comment by mtgsquad** [comment]
Hey, if you want to index your site on google, please use the meta tag verification, then you can simply add it to your sites HTML :)

**Pull request closed without merging**